flake.nix

```diff
@@ -1,124 +1,26 @@
-        PYTHON = "${pkgs.python3}/bin/python3";
-        # chromedriver wants to download the binary
-        CHROMEDRIVER_FILEPATH = "${chromedriverBin}";
-        # node-hid uses pkg-config to find sources
-        buildInputs = oldAttrs.buildInputs ++ [pkgs.pkg-config pkgs.libusb1];
-        # run actual build
-        buildPhase = ''
-          yarn workspace @cardano-sdk/cardano-services run build
-        '';
-        # override installPhase to only install what's necessary
-        installPhase = ''
-          runHook preInstall
-
-          mkdir -p $out/libexec/$sourceRoot $out/bin
-          yarn workspace @cardano-sdk/cardano-services-client run build:version
-
-          cp -r ${production-deps}/libexec/$sourceRoot/node_modules $out/libexec/$sourceRoot/node_modules
-          cp -r scripts $out/libexec/$sourceRoot/scripts
-          for p in cardano-services cardano-services-client core ogmios util crypto projection projection-typeorm util-rxjs; do
-            mkdir -p $out/libexec/$sourceRoot/packages/$p
-            cp -r packages/$p/dist $out/libexec/$sourceRoot/packages/$p/dist
-            cp -r packages/$p/package.json $out/libexec/$sourceRoot/packages/$p/package.json
-          done
-          cp -r ${production-deps}/libexec/$sourceRoot/packages/cardano-services/config $out/libexec/$sourceRoot/packages/cardano-services/config
-
-          cd "$out/libexec/$sourceRoot"
+  inputs = {
+    std.url = "github:divnix/std"; # keep minor version in lockstep with GH std-action
+    # std.inputs.nixpkgs.follows = "nixpkgs";
+    std.inputs.n2c.follows = "n2c";
+    haumea.url = "github:nix-community/haumea/v0.2.2";
+    n2c.url = "github:nlewo/nix2container";
+    n2c.inputs.nixpkgs.follows = "std/nixpkgs";
+  };
 
-          runHook postInstall
-        '';
-        # add a bin script that should be used to run cardano-services CLI
-        postInstall = ''
-          cat > $out/bin/cli <<EOF
-          #!${pkgs.bash}/bin/bash
-          exec "${pkgs.nodejs}/bin/node" "$out/libexec/$sourceRoot/packages/cardano-services/dist/cjs/cli.js" "\$@"
-          EOF
-          chmod a+x $out/bin/cli
-        '';
-        meta.mainProgram = "cli";
-        passthru.nodejs = pkgs.nodejs;
-        passthru.production-deps = production-deps;
-      });
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixpkgs-unstable";
 
-    apps = flake-utils.lib.eachDefaultSystemMap (system: let
-      pkgs = nixpkgs.legacyPackages.${system};
-    in {
-      config-update = flake-utils.lib.mkApp {
-        drv = pkgs.writeShellApplication {
-          name = "config-update";
-          runtimeInputs = with pkgs; [git git-subrepo];
-          text = ''
-            git subrepo pull packages/cardano-services/config --message='chore: git subrepo pull packages/cardano-services/config' "$@"
-          '';
-        };
-      };
-    });
-  };
+  outputs = {std, ...} @ inputs:
+    std.growOn {
+      inherit inputs;
+      cellsFrom = ./ops;
```

Comment:
Consider changing this folder to `nix` or something else. As a third-party contributor it would take me a little to figure out that all this magic is hiding in this folder

Reply:
It was discussed as an option to better capture the intent but that was a different context. Here we have more than just ops code, as the lower packaging layers are present, so it would be more aligned as `nix`

Reply:
Addressed in 93cd451b3bf08ad132da0e18839473efda672658